src/main/java/it/pagopa/pagopa/apiconfig/ApiConfig.java

```diff
@@ -5,12 +5,15 @@
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 import org.springframework.retry.annotation.EnableRetry;
 
+import java.util.Locale;
+
 @SpringBootApplication
 @EnableJpaRepositories
 @EnableRetry
 public class ApiConfig {
 
     public static void main(String[] args) {
+        Locale.setDefault(Locale.ENGLISH);
```

Comment:
why added it ?

Reply:
in the autogenerated response of bad request, the message was in Italian